Tools/llm-tool/LLMTool.swift

```diff
@@ -131,6 +133,8 @@ struct AsyncGenerator: AsyncParsableCommand {
 
         let modelConfiguration = ModelConfiguration.configuration(id: model)
         let (model, tokenizer) = try await load(configuration: modelConfiguration)
+        
+        print("Model loaded -> \(self.model)")
```

Comment:
These look like they might be for debugging -- do you mean to leave them in here?

Reply:
Yes. There are many models configured in Scheme. Sometimes I forget which model is currently being used, so I want to display the currently loaded model.

![image](https://github.com/ml-explore/mlx-swift-examples/assets/1825679/2194bf84-d854-4c7c-97c1-0d0e4a0a6bc6)